src/api/app/views/webui2/webui/package/binary.html.haml

```diff
@@ -0,0 +1,51 @@
+- @pagetitle = "Detailed Information About #{@filename}"
+- @metarobots = 'noindex' # files change too often
+
+.card
+  = render partial: 'tabs', locals: { package: @package, project: @project }
+
+  .card-body
+    %h3
+      Detailed Information About
```

Comment:
This should be `@pagetitle` to be consistent. See https://github.com/openSUSE/open-build-service/wiki/How-to-work-in-Bootstrap-(new-UI)#implementation-details

Reply:
On second thought... the header contains a link in some cases. IMO this doesn't make sense in the page title.
@dmarcoux Any idea how we can handle this?

Reply:
Oh, if it's not possible then, I would just leave it as it is.

Reply:
I found a solution :bowtie:  


![](https://screenshotscdn.firefoxusercontent.com/images/6e31a67d-4d00-4aba-b973-774d44c692d9.png)